### PR TITLE
[Bluetooth] Add BluetoothHid APIs - initial commit

### PR DESCRIPTION
**Pull request by Wootak**
- Interop
- BluetoothEnumerations
- BluetoothData

### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothData.cs [hunk 1]
```diff
@@ -542,4 +542,151 @@ public string ServiceUuid
...
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// The key value - currently pressed keys : Max 8 at once
+        /// </summary>
+        /// <since_tizen> 5.5 </since_tizen>
+        public byte[] Key
```
[the remark this answers is not in the document]
Comment (myroot) [reply src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothData.cs]: Why not string?

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothData.cs [hunk 2]
```diff
@@ -542,4 +542,151 @@ public string ServiceUuid
...
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// The received data
+        /// </summary>
+        /// <since_tizen> 5.5 </since_tizen>
+        public string Data
```
Comment (myroot) [inline src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothData.cs]: Why use string?
Why not byte[]?

File: src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothData.cs [hunk 3]
```diff
@@ -542,4 +542,151 @@ public string ServiceUuid
...
+        {
+            get;
+            set;
+        }
+
+        /// <summary>
+        /// The length of the received data
+        /// </summary>
+        /// <since_tizen> 5.5 </since_tizen>
+        public int DataSize
```
Comment (myroot) [inline src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothData.cs]: no need in C#

**Comment by Wootak** [comment]
I will submit a new PR and close this PR.

**Pull request closed without merging**